unattended_installer/install_functions/opendistro/common.sh

```diff
@@ -176,7 +181,7 @@ function readPasswordFileUsers() {
 
     filecorrect=$(grep -Pzc '\A(User:\s*name:\s*\w+\s*password:\s*[A-Za-z0-9_\-]+\s*)+\Z' "${p_file}")
     if [ "${filecorrect}" -ne 1 ]; then
-        logger_pass -e "The password file doesn't have a correct format.
+        logger -e "The password file doesn't have a correct format.
```

Comment:
Are you sure that `logger` must be used instead of `logger_pass`? It could look the same, but the log file is different.

Reply:
Yes, this was left behind after copying the function from wazuh_passwords_tool, but logger_pass may not be defined when calling the functions in common.sh.